allofplos/plos_corpus.py

```diff
@@ -777,19 +798,20 @@ def main():
                         'Used when inside the plos network')
     args = parser.parse_args()
     plos_network = False
+    directory = get_corpus_dir()
     if args.plos:
         URL_TMP = INT_URL_TMP
         plos_network = True
     else:
         URL_TMP = EXT_URL_TMP
     # Step 0: Initialize first copy of repository
     try:
-        corpus_files = [name for name in os.listdir(corpusdir) if os.path.isfile(
-                        os.path.join(corpusdir, name))]
+        corpus_files = [name for name in os.listdir(directory) if os.path.isfile(
+                        os.path.join(directory, name))]
     except FileNotFoundError:
         corpus_files = []
     if len(corpus_files) < min_files_for_valid_corpus:
-        print('Not enough articles in corpusdir, re-downloading zip file')
+        print('Not enough articles in {}, re-downloading zip file').format(directory)
```

Comment:
Probably don't want to insert the whole file path here, I'd just say "specified directory"

Reply:
I actually don't agree about that one… suppose someone is actually trying to get this from the wrong directory; this is going to be the best opportunity we have to communicate that information to them.